src/odemis/odemisd/start.py

```
@@ -352,12 +352,13 @@ def start_backend(self, modelfile):
         """
         # install cgroup, for memory protection
-        if (
-            not os.path.exists("/sys/fs/cgroup/memory/odemisd") and
-            os.path.exists("/usr/bin/cgcreate")
-        ):
+        if os.path.exists("/usr/bin/cgcreate"):
             logging.info("Creating cgroup")
-            subprocess.call(["sudo", "/usr/bin/cgcreate", "-a", ":odemis", "-g", "memory:odemisd"])
+            # Note: there should be a sudoers rule allowing to run this command without password
+            try:
+                subprocess.check_call(["sudo", "-n", "/usr/bin/cgcreate", "-a", ":odemis", "-g", "memory:odemisd"])
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)